components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_state.cc

```diff
@@ -98,4 +100,16 @@ bool IsNetworkFiltersInstalled() {
   return current > 0;
 }
 
+// The service starts under sytem user so we save crashes to
```

Comment:
how do we differentiate between crashes from different users on the same machine who may have different crash upload settings?

Reply:
Also, `sytem user` typo.

Reply:
`crash upload settings` is working via https://source.chromium.org/chromium/chromium/src/+/main:chrome/install_static/install_util.cc;l=413?q=GetCollectStatsConsent&ss=chromium%2Fchromium%2Fsrc